### PR TITLE
Deploy minimal domain events consumer to production

### PR DESCRIPTION
**Pull request by ayqazi**
MO-1360

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -97,7 +97,7 @@ class Application < Rails::Application
 
     config.domain_event_handlers = {
       'offender-management.noop' => 'DomainEvents::Handlers::NoopHandler',
-      'prisoner-offender-search.prisoner.updated' => 'DomainEvents::Handlers::PrisonerUpdatedHandler',
+      # 'prisoner-offender-search.prisoner.updated' => 'DomainEvents::Handlers::PrisonerUpdatedHandler',
```
Comment (SebNegahbanMoJ) [inline config/application.rb:100]: I assume this is an intentional commenting-out?

Comment (ayqazi) [reply config/application.rb:100]: Yep, I'll refer you to the commit where the change was made for the explanation

**Review by SebNegahbanMoJ — APPROVED** [review]
I can't claim to fully understand the config but it all looks reasonable!

**Comment by ayqazi** [comment]
> I can't claim to fully understand the config but it all looks reasonable!

I copied something that already works 🤷🏾

**Pull request merged**